ntbs-service/Models/Entities/DataQualityBirthCountryAlert.cs

```diff
@@ -1,10 +1,18 @@
-﻿using ntbs_service.Helpers;
+﻿using System;
+using System.Linq.Expressions;
+using ntbs_service.Helpers;
 using ntbs_service.Models.Enums;
 
 namespace ntbs_service.Models.Entities
 {
     public class DataQualityBirthCountryAlert : Alert
     {
+        public static readonly Expression<Func<Notification, bool>> NotificationQualifiesExpression = 
+            n => n.PatientDetails.CountryId == Countries.UnknownId;
+
+        public static readonly Func<Notification, bool> NotificationQualifies =
```

Comment:
Slight annoying suggestion, but I think NotificationQualifies in some contexts could be ambigious, as in does it qualify to not have the alert or does it qualify for the alert.
I think `IsNotificationEligibleForAlert` might fit the bill less ambigiously - definitely happy to be overruled in this. When considering this point I flip-flopped a couple of times.

Reply:
I can see the worry, but I think importantly the way this is consumed is *always* through
`DataQualityBirthCountryAlert.NotificationQualifies` (etc.), which I think gives it enough context? I feel it'd be pretty odd if *that* was trying to say "notifcation qualifies for no longer needing the alert"?